requirements.txt

```diff
@@ -5,4 +5,7 @@ openpyxl
 plotly
 pyopenephys>=1.1.6
 seaborn
-scikit-image
\ No newline at end of file
+scikit-image
+spikeinterface
+spikeextractors
```

Comment:
```suggestion
```

Reply:
I could be mistaken, but I don't believe that `spikeextractors` is required for this package.